jest.config.js

```diff
@@ -1,4 +1,5 @@
 const FULL_TEST = Boolean(process.env.FULL_TEST);
+const TEST_STANDALONE = Boolean(process.env.TEST_STANDALONE);
```

Comment:
where this env variable will be used? I cannot see it in this PR

Reply:
this variable is used to run tests.
We don't actively use in our automated tests right now but the idea is to have an environment that uses only a browser friendly environment.